src/Rollbar.php

```diff
@@ -107,49 +146,49 @@ public static function logUncaught($level, Throwable $toLog, $extra = array())
             return self::getNotInitializedResponse();
         }
         $toLog->isUncaught = true;
-        $result = self::$logger->log($level, $toLog, (array)$extra);
+        $result = self::$logger->report($level, $toLog, (array)$extra);
         unset($toLog->isUncaught);
```

Comment:
What do you think about wrapping this block in a try-finally to make sure we always unset `isUncaught`?

Reply:
That is a really solid idea. Otherwise, it would be possible for our dynamic property to persisting if the `Throwable` was thrown in `report()`.

I have committed that change.